Ceritas-Team/ceritas-api
# Core Organizations Route & Resolvers

## Description

This pull request adds a new route to the GraphQL API that allows clients to query`coreOrganizations` data. The new route is defined in the schema, with corresponding resolvers defined in the `queries/core_organization` module. The `CoreOrganization` model and corresponding models have been updated to support the new route.

## Changes Made

- Added new `coreOrganizations` route to the GraphQL API
- Defined corresponding resolvers in `queries/core_organization`
- Updated `CoreOrganization` model with new fields and relationships

## How to Test

1. Check out this branch: `git checkout dev-lucas`
2. Install dependencies: `pip install`
3. Start the development server: `flask run`
4. Query the new `coreOrganizations` route and verify that it returns expected results
